### ISSUE TITLE
RCS1077: Collections returned from a function call are not checked

### ISSUE THREAD

**Issue by JoshuaLight** [JosefPihrt/Roslynator#668]
**Product and Version Used**:
`Roslynator.Analyzers 2.3.0`.

**Steps to Reproduce**:
The following listing of code demonstrates that the diagnostic is not reported when the expression contains a function call instead of a property call or a simple access.
```csharp
static List<int> Func() => new List<int>();
static List<int> Property => new List<int>();

// ...

var a = Func().First();     // (A)
var b = Property.First();   // (B)
```

**Actual Behavior**:
The expression _B_ is reported, the expression _A_ isn't.

**Expected Behavior**:
The expression _B_ is reported, the expression _A_ is reported as well.

**Comment by JosefPihrt** [comment]
Fixed with ffb3b97c4e4e8ddb018be006ce240236c0fd82e1

Related issue: https://github.com/JosefPihrt/Roslynator/issues/654
